java/bogdana_maksiuta/src/com/zoolatech/lecture1/tasks/_2/SpeedCalculator.java

```diff
@@ -0,0 +1,42 @@
+package com.zoolatech.lecture1.tasks._2;
+
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts a distance in meters and time in seconds and outputs
+ * a speed in meters/second and kilometers/hour. The result should be a floating point number.
+ *         a. Input: 80
+ *           10
+ *           Output: 8.0
+ * 	         28.8
+ */
+public class SpeedCalculator {
+    public static void main(String[] args) {
+        System.out.println("Введіть відстань: ");
+        Scanner scanner = new Scanner(System.in);
+        float distance = scanner.nextFloat();
+//        float distance = 80; //for testing
+        System.out.println("Введіть час: ");
+        Scanner scanner1 = new Scanner(System.in);
```

Comment:
It's possible to use a single instance of the scanner class, you don't need to create a second one.

Reply:
fixed